fulp_modules/features/species/protogen/protogen.dm

```diff
@@ -0,0 +1,78 @@
+/datum/species/protogen
+	name = "Protogen"
+	plural_form = "Protofriends"
+	id = SPECIES_PROTOGEN
+	examine_limb_id = SPECIES_PROTOGEN
+	mutant_bodyparts = list(
+		"legs" = "Normal Legs",
+	)
+	external_organs = list(
+		/obj/item/organ/external/snout/protogen = "Bolted",
+		/obj/item/organ/external/tail/protogen = "Shark",
+		/obj/item/organ/external/protogen_antennae = "Default",
+	)
+	inherent_traits = list(
+		TRAIT_MUTANT_COLORS,
+		TRAIT_TACKLING_TAILED_DEFENDER,
+	)
+	inherent_biotypes = MOB_ROBOTIC|MOB_HUMANOID
+	meat = /obj/item/food/meat/slab/human/mutant/lizard
+	mutantbrain = /obj/item/organ/internal/brain/cybernetic
+	mutanttongue = /obj/item/organ/internal/tongue/robot
+	mutantstomach = /obj/item/organ/internal/stomach/cybernetic/surplus
+	mutantappendix = null
+	mutantheart = /obj/item/organ/internal/heart/cybernetic/surplus
+	mutantliver = /obj/item/organ/internal/liver/cybernetic/surplus
+	mutantlungs = /obj/item/organ/internal/lungs/cybernetic/surplus
+	mutanteyes = /obj/item/organ/internal/eyes/robotic
+	mutantears = /obj/item/organ/internal/ears/cybernetic
+	species_language_holder = /datum/language_holder/synthetic
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT
```

Comment:
If this is supposed to be admin-only / april fools-only then you probably shouldn't let players transform into them no?